Xamarin.Forms.Platform.Android/AppCompat/CarouselPageRenderer.cs

```diff
@@ -35,6 +37,14 @@ void ViewPager.IOnPageChangeListener.OnPageScrollStateChanged(int state)
 
 		IPageController PageController => Element as IPageController;
 
+		FragmentManager FragmentManager => _fragmentManager ?? (_fragmentManager = ((FormsAppCompatActivity)Context).SupportFragmentManager);
```

Comment:
```suggestion
		FragmentManager FragmentManager => _fragmentManager ?? (_fragmentManager = Context.GetFragmentManager());
```

Reply:
I tried to just commit this myself but looks like I don't have permissions on the repository.